index.js

```diff
@@ -141,6 +144,12 @@ File.prototype.inspect = function() {
   return '<File ' + inspect.join(' ') + '>';
 };
 
+File.builtInFields = ['_contents', 'contents', 'stat', 'history', 'path', 'base', 'cwd'];
```

Comment:
I don't think this should be exposed on the constructor.  Not sure of a better way to do it.